setup.py

```diff
@@ -11,7 +11,7 @@
 requirements = [ 'Distroinfo>=0.1',
                  'git_wrapper>=0.2.2' ]
 
-test_requirements = [ 'mock', 'pytest', ]
+test_requirements = [ 'mock', 'pytest', 'flake8', 'pyest-cov',]
```

Comment:
Hm... clearly this works, but shouldn't this be "pytest-cov"? I think it comes from the test-requirements.txt file in the tests anyway, so perhaps this file doesn't need the update.

Reply:
Good catch, I am not used to this being duplicated in the requirements file anymore, since our other projects don't do it that way. I'll just remove these references for now, we can modernize the packaging later/separately.